src/utils/scripts/data.csv

```diff
@@ -0,0 +1,4 @@
+Question,Answear1,Points,1,Answear2,Points2,Answear3,Points3,Answear4,Points,4
```

Comment:
In this template, let's include all 8 answers and points. Also, be careful of where you place the comas, as in this example, the `1` from `Points1` will be a column.
We should create this template just as it should be used, in order to send it to the event organizers.